src/main/java/de/hysky/skyblocker/config/SkyblockerConfig.java

```diff
@@ -602,6 +602,9 @@ public static class Dungeons {
 		@SerialEntry
 		public DungeonChestProfit dungeonChestProfit = new DungeonChestProfit();
 
+		@SerialEntry
+		public MimicMessages mimicMessages = new MimicMessages();
```

Comment:
For future reference use singular for consistency.

Reply:
That name is fine, no problem with the duplicate. Do you mind changing it to that?